src/blocks/temperature.rs

```diff
@@ -93,7 +95,7 @@ pub struct TemperatureConfig {
 
 impl TemperatureConfig {
     fn default_format() -> String {
-        "{average}° avg, {max}° max".to_owned()
```

Comment:
Is the degrees symbol deleted for a reason?

Reply:
Yes. Every placeholder displays a unit on it's own.

Reply:
Oh right, sorry about that!

Reply:
Then the docs need updating too since it still has the degrees symbol in the example config and in the format string description.